presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -225,6 +228,20 @@ public SystemSessionProperties(
                         COLOCATED_JOIN,
                         "Experimental: Use a colocated join when possible",
                         featuresConfig.isColocatedJoinsEnabled(),
+                        false),
+                new PropertyMetadata<>(
+                        OPERATOR_MEMORY_LIMIT_BEFORE_SPILL,
+                        "Experimental: Operator memory limit before spill",
```

Comment:
I think it's more complicated then that. It should be explained somewhere (in tuning guide?), but not here.